client/src/components/text-input/text-input.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@/components/Component';
+
+import '@/styles/components/text-input/text-input.scss';
```

Comment:
```suggestion
import styles from '@/styles/components/text-input/text-input.module.scss';
```

#55 에서 언급한 것과 같이 module.scss를 사용해주시기 바랍니다.

Reply:
넵. 수정했습니다. 다른 부분들도 그렇게 수정하겠습니다.